src/constants.ts

```diff
@@ -15,6 +15,7 @@ export const DEFAULT_LOGGER_INCLUDE_TIMESTAMP = true;
 export const ONE_SECOND_MS = 1000;
 export const ONE_MINUTE_MS = ONE_SECOND_MS * 60;
 export const TEN_MINUTES_MS = ONE_MINUTE_MS * 10;
+export const GST2_TOKEN_ADDRESS = '0x0000000000b3f879cb30fe243b4dfee438691c04';
```

Comment:
Might this belong in the @0x/contract-addresses package? What happens on kovan / ganache?

Reply:
The contract-addresses package mostly contains smart contracts that we own, this is just a token, so put it in the token_metadatas_for_networks file.